02-starting-with-data.Rmd

```diff
@@ -123,6 +245,11 @@ useful but not necessarily intuitive, and therefore require some attention.
 ### Factors
 ```
 
+When we did `str(surveys)` we see that several of the columns consist of
+integers, however, the columns `genus`, `species`, `sex`, `plot_type`, ... are
+of a special class called a `factor`. Factors are very useful but not
+necessarily intuitive, and therefore require some attention.
+
```

Comment:
suggest

Factor are very useful and are actually something that make R particularly well suited to working with data, so we're going to spend a little time introducing them.